java/arcs/core/data/proto/annotation.proto

```diff
@@ -6,12 +6,17 @@ option java_package = "arcs.core.data.proto";
 option java_multiple_files = true;
 
 // Represents the usage of an annotation.
-// TODO(b/155796088): Add support for all annotation functionality, including
-// annotation definitions and all parameter types.
-message AnnotationRefProto {
+message AnnotationProto {
```

Comment:
Is this intended to represent the usage of an annotation, or the definition of an annotation?

I'd intended the former, and called it `AnnotationRefProto` to be consistent with how that is named in TS. Were you ever intending to store annotation definitions in protos?

Reply:
This is intended to be the usage of an annotation (aka AnnotationRef in ts).
However the data members (in TS also) are called annotations, and I think it is more readable if the proto is called annotation as well (we in general don't use all the exact same names in the kt classes and protos and optimize for the reader).

I don't expect that we'd send the annotations declarations to KT (not anytime soon at least).

PS: If I were to rename anything, I'd rename the ts manifest's Annotation to AnnotationDeclaration and AnnotationRef to Annotation.

Reply:
That sounds reasonable to me, thanks!